gcc/rust/rust-session-manager.h

```diff
@@ -203,6 +204,13 @@ struct CompileOptions
     enable_dump_option (DumpOption::HIR_DUMP);
     enable_dump_option (DumpOption::TYPE_RESOLUTION_DUMP);
   }
+
+  bool set_crate_name (std::string name)
```

Comment:
is there a getter for this? Not sure if we need a move here. I would have either done:

```
bool set_crate_name (std::string&& name) { crate_name = std::move(name); }
```

Or copy

```
void set_crate_name(const std::string& name) { crate_name.assign(name); }
```

Reply:
It is public already (`CompileOptions` is a `struct`).